main_test.go

```diff
@@ -72,14 +72,14 @@ func TestMain(t *testing.T) {
 	// Start main!
 	go main()
 
-	// Create json for request
-	json, err := json.Marshal(volume.Request{})
+	// Create json for request - local variable json masks the global symbol json referring to the JSON module
+	jsn, err := json.Marshal(volume.Request{})
```

Comment:
ah, yes that makes sense
I wonder how this didn't cause problems before in our testing?